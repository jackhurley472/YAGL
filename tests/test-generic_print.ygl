Node n("hi");
print(n);
int x = 5;
print(x);
char s = 's';
print(s);
print('s');
Graph g;
g: + n;
print(g);
print("Hi!!!");
print(true);
