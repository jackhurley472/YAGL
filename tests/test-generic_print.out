hi
5
s
s
============== Graph Print ===============
	Stats: 	1	1	0

Nodes:
Node (0) : hi

Edges:
There aren't any edges in this graph.
============ End Graph Print =============
Hi!!!
1
